kolibri/core/content/test/test_import_export.py

```diff
@@ -179,7 +179,7 @@ def test_remote_import_readtimeout(
 
 
 @patch(
-    "kolibri.core.content.management.commands.importcontent.calculate_files_to_transfer",
+    "kolibri.core.content.management.commands.importcontent.calculate_files_to_transfer"
```

Comment:
not a big deal, but strange that this was necessary ... are you running the latest version of Black? I know we had some annoying version inconsistency issues recently